oci/defaults.go

```diff
@@ -37,7 +32,7 @@ func DefaultWindowsSpec() specs.Spec {
 
 // DefaultLinuxSpec create a default spec for running Linux containers
 func DefaultLinuxSpec() specs.Spec {
-	s := specs.Spec{
+	return specs.Spec{
```

Comment:
Changes in this file are easiest reviewed with "ignore whitespace"; https://github.com/moby/moby/pull/42683/files?w=1